docs/userGuide/syntax/textStyles.md

```diff
@@ -26,10 +26,14 @@ Syntax added by MarkBind:
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">markdown</variable>
 <variable name="code">
-****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--, Super^script^, Sub~script~
+****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--, Super^script^, Sub~script~, #r#Coloured Text##
```

Comment:
On the website this line is getting too long and being hidden for smaller screen, could we split the lines into two, so perhaps
```
****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--

Super^script^, Sub~script~, #r#Coloured Text##

```